### PR TITLE
Update drone test intsructions in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by techknowlogick**
Fix #4053
Fix #3977

Credits: @axifive

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ running the test suite exactly like we do, you should install
 the CLI for [Drone CI](https://github.com/drone/drone), as
 we are using the server for continous testing, following [these
 instructions](http://readme.drone.io/usage/getting-started-cli). After that,
-you can simply call `drone exec` within your working directory and it will try
+you can simply call `drone exec --local --build-event "pull_request"` within your working directory and it will try
```
Comment (stevegt) [inline CONTRIBUTING.md]: Should be `--build.event`

Comment (techknowlogick) [reply CONTRIBUTING.md]: @stevegt per the drone help output from cli it's `--build-event`.

See:
```
NAME:
   drone exec - execute a local build

USAGE:
   drone exec [command options] [path/to/.drone.yml]

OPTIONS:
   --local                            build from local directory [$DRONE_LOCAL]
   --timeout value                    build timeout (default: 1h0m0s) [$DRONE_TIMEOUT]
   --volumes value                    build volumes [$DRONE_VOLUMES]
   --network value                    external networks [$DRONE_NETWORKS]
   --privileged value                 privileged plugins (default: "plugins/docker", "plugins/gcr", "plugins/ecr")
   --workspace-base value             (default: "/drone") [$DRONE_WORKSPACE_BASE]
   --workspace-path value             (default: "src") [$DRONE_WORKSPACE_PATH]
   --netrc-username value              [$DRONE_NETRC_USERNAME]
   --netrc-password value              [$DRONE_NETRC_PASSWORD]
   --netrc-machine value               [$DRONE_NETRC_MACHINE]
   --system-arch value                (default: "linux/amd64") [$DRONE_SYSTEM_ARCH]
   --system-name value                (default: "pipec") [$DRONE_SYSTEM_NAME]
   --system-link value                (default: "https://github.com/cncd/pipec") [$DRONE_SYSTEM_LINK]
   --repo-name value                   [$DRONE_REPO_NAME]
   --repo-link value                   [$DRONE_REPO_LINK]
   --repo-remote-url value             [$DRONE_REPO_REMOTE]
   --repo-private value                [$DRONE_REPO_PRIVATE]
   --build-number value               (default: 0) [$DRONE_BUILD_NUMBER]
   --parent-build-number value        (default: 0) [$DRONE_PARENT_BUILD_NUMBER]
   --build-created value              (default: 0) [$DRONE_BUILD_CREATED]
   --build-started value              (default: 0) [$DRONE_BUILD_STARTED]
   --build-finished value             (default: 0) [$DRONE_BUILD_FINISHED]
   --build-status value                [$DRONE_BUILD_STATUS]
   --build-event value                 [$DRONE_BUILD_EVENT]
   --build-link value                  [$DRONE_BUILD_LINK]
   --build-target value                [$DRONE_BUILD_TARGET]
   --commit-sha value                  [$DRONE_COMMIT_SHA]
   --commit-ref value                  [$DRONE_COMMIT_REF]
   --commit-refspec value              [$DRONE_COMMIT_REFSPEC]
   --commit-branch value               [$DRONE_COMMIT_BRANCH]
   --commit-message value              [$DRONE_COMMIT_MESSAGE]
   --commit-author-name value          [$DRONE_COMMIT_AUTHOR_NAME]
   --commit-author-avatar value        [$DRONE_COMMIT_AUTHOR_AVATAR]
   --commit-author-email value         [$DRONE_COMMIT_AUTHOR_EMAIL]
   --prev-build-number value          (default: 0) [$DRONE_PREV_BUILD_NUMBER]
   --prev-build-created value         (default: 0) [$DRONE_PREV_BUILD_CREATED]
   --prev-build-started value         (default: 0) [$DRONE_PREV_BUILD_STARTED]
   --prev-build-finished value        (default: 0) [$DRONE_PREV_BUILD_FINISHED]
   --prev-build-status value           [$DRONE_PREV_BUILD_STATUS]
   --prev-build-event value            [$DRONE_PREV_BUILD_EVENT]
   --prev-build-link value             [$DRONE_PREV_BUILD_LINK]
   --prev-commit-sha value             [$DRONE_PREV_COMMIT_SHA]
   --prev-commit-ref value             [$DRONE_PREV_COMMIT_REF]
   --prev-commit-refspec value         [$DRONE_PREV_COMMIT_REFSPEC]
   --prev-commit-branch value          [$DRONE_PREV_COMMIT_BRANCH]
   --prev-commit-message value         [$DRONE_PREV_COMMIT_MESSAGE]
   --prev-commit-author-name value     [$DRONE_PREV_COMMIT_AUTHOR_NAME]
   --prev-commit-author-avatar value   [$DRONE_PREV_COMMIT_AUTHOR_AVATAR]
   --prev-commit-author-email value    [$DRONE_PREV_COMMIT_AUTHOR_EMAIL]
   --job-number value                 (default: 0) [$DRONE_JOB_NUMBER]
```

Comment (stevegt) [reply CONTRIBUTING.md]: CONTRIBUTING.md sends the user to 
http://readme.drone.io/usage/getting-started-cli for drone install instructions.  That page then tells linux users to do this:

<pre>
curl http://downloads.drone.io/release/linux/amd64/drone.tar.gz | tar zx
sudo install -t /usr/local/bin drone
</pre>

That currently installs drone version 0.5.0+dev, which says this in its help output:

<pre>
   --build.event "push"                                         build event [$DRONE_BUILD_EVENT]
</pre>

Which version are you running?

Comment (techknowlogick) [reply CONTRIBUTING.md]: I'm using 0.8.5 which is the same as Gitea's (the project) version of drone that runs all PRs.

Drone has out of date info on the "readme" site, and you'll want to refer to docs.drone.io for up to date info

Here is a more recent set of instructions on how to install drone on linux
```
curl -L https://github.com/drone/drone-cli/releases/download/v0.8.5/drone_linux_amd64.tar.gz | tar zx
sudo install -t /usr/local/bin drone
```

Comment (stevegt) [reply CONTRIBUTING.md]: Arg.  Okay, just found http://docs.drone.io/cli-installation/, which instead says:

<pre>
curl -L https://github.com/drone/drone-cli/releases/download/v0.8.5/drone_linux_amd64.tar.gz | tar zx
sudo install -t /usr/local/bin drone
</pre>

Comment (stevegt) [reply CONTRIBUTING.md]: Created https://github.com/go-gitea/gitea/pull/4071

**Pull request closed without merging**